### ISSUE TITLE
Capture DOM changes

### ISSUE THREAD

**Issue by hamzazar** [orthes/medium-editor-insert-plugin#212]
Hi All,

How can I capture the modification on contenteditable when editing the image, delte or grid button for exemple.

this method below as described on medium-editor docs not be fired with image insert plugin toolbar but it fire when the contenteditable change from clicking on buttons in the toolbar 

```js
var editor = new MediumEditor('.editable');
editor.subscribe('editableInput', function (event, editable) {
    // Do some work
});
```

**Comment by orthes** [comment]
Try this:

```javascript
editor.subscribe('editableInput, input', function (event, editable) {
    // Do some work
});
```

Apparently, MediumEditor changed their event from ```input``` to ```editableInput``` and we haven't caught up yet. It should just be a proxy for the ```input```, so I hoped it would work just fine without any changes.

**Comment by hamzazar** [comment]
Thanks for the suggestion but changing from 'editableInput' to 'editableInput, input' didn't fix the problem and the medium editor toolbar won't fire the function anymore..

**Comment by orthes** [comment]
Yes, I fixed the problem in #216 but we have some problems with tests on that PR. Once we merge it, we'll release a new version with the fix.
